### PR TITLE
Feat/mintsquare-mainnet support

### PR DESCRIPTION
**Pull request by dhruvkelawala**
Add Mintsquare’s Mainnet URL for NFTs

`https://mintsquare.io/asset/starknet/{contract_address}/{token_id}`

### CODE REVIEW

File: packages/extension/src/ui/features/accountNfts/mint-square.service.ts [hunk 1]
```diff
@@ -1,11 +1,18 @@
 import { number } from "starknet"
 
-export const openMintSquareNft = (contractAddress: string, tokenId: string) => {
+export const openMintSquareNft = (
+  contractAddress: string,
+  tokenId: string,
+  networkId: string,
```
Comment (janek26) [inline packages/extension/src/ui/features/accountNfts/mint-square.service.ts:6]: lets catch the case where networkId is not main net nor goerli? like localhost

File: packages/extension/src/ui/components/ErrorBoundaryFallbackWithCopyError.tsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ const ErrorMessageContainer = styled.div`
   margin-bottom: 16px;
 `
 
-export const ActionsWrapper = styled.div`
+const ActionsWrapper = styled.div`
```
Comment (janek26) [inline packages/extension/src/ui/components/ErrorBoundaryFallbackWithCopyError.tsx:44]: is this expected?

Comment (dhruvkelawala) [reply packages/extension/src/ui/components/ErrorBoundaryFallbackWithCopyError.tsx:44]: This is part of the previous PR

https://github.com/argentlabs/argent-x/pull/1019

File: packages/extension/src/ui/features/accountNfts/mint-square.service.ts (hunk 1 above)
Comment (dhruvkelawala) [reply packages/extension/src/ui/features/accountNfts/mint-square.service.ts:6]: Then we should do it for Aspect as well, no?

Comment (janek26) [reply packages/extension/src/ui/features/accountNfts/mint-square.service.ts:6]: I think we do!

**Review by janek26 — APPROVED** [review]

**Pull request merged**